src/midi/midi_fluidsynth.cpp

```diff
@@ -192,15 +175,40 @@ static std::deque<std_fs::path> get_data_dirs()
 
 #endif
 
-static std_fs::path find_sf_file(const std::string& name)
+static Section_prop* get_fluidsynth_section()
 {
-	const std_fs::path sf_path = resolve_home(name);
+	assert(control);
+
+	auto sec = static_cast<Section_prop*>(control->GetSection("fluidsynth"));
+	assert(sec);
+
+	return sec;
+}
+
+static std::vector<std_fs::path> get_data_dirs()
+{
+	auto dirs = get_platform_data_dirs();
+
+	auto sf_dir = get_fluidsynth_section()->Get_string("soundfont_dir");
+	if (!sf_dir.empty()) {
+		// The user-provided SoundFont dir might use a different casing
+		// of the actual path on Linux & Windows, so we need to
+		// normalise that to avoid some subtle bugs downstream (see
+		// `find_sf_file()` as well).
+		dirs.push_back(std_fs::canonical(sf_dir));
+	}
```

Comment:
Maybe give the user's directory first billing by putting it first ` dirs.insert(dirs.begin(), ...)`

Reply:
Ah, that's my mistake. That's what I _wanted_ to do and that's what I said in the release notes. Good catch! 🎖️

Reply:
Done.